.github/workflows/e2e_tests.yml

```diff
@@ -112,60 +118,8 @@ jobs:
         id: test-playwright
         # No Playwright tests in LWC 3.5
         if: ${{ env.BRANCH != 'release_3_5' }}
-        run: cd end2end && npx playwright test --project=chromium
+        run: cd end2end && npx playwright test --project=${{ matrix.project }} --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }}
 
-      - name: Notify in case of playwright failure, from mainstream branches only
```

Comment:
Are you sure you want to remove this code ? We won't have any notifications when it's failing in the cronjob.

Reply:
I removed many things just to test quickly.